client/stylesheets/_toCat.scss

```diff
@@ -3499,8 +3586,95 @@ On click the images will be displayed at normal size to complete the effect
     margin: 0px 0 0 10px;
     vertical-align: middle;
     color: #fff !important;
+} */
+
+.profileChatBtn a {
+    font-size: 1.2rem;
+    padding: 15px 22px;
+    color: #fff !important;
+    background-color: #f8504b;
+    border: 1px solid #f8504b;
+    box-shadow: 0 -1px 0 rgba(255, 255, 255, 0.1) inset;    
+    border-radius: 6px;
+    overflow: hidden;
 }
 
+.profileChatBtn a:hover {
+    background-color: #f73c37;
+}
+
+
+.optionsProClick {
+	margin-left: 10px;
+}
+
+.optionsProClick ul {
+	position:relative;
+	font-weight:400;
+}
+
+.optionsProClick ul li {
+	position:relative;
+	list-style:none;
+	cursor:pointer;
+}
+
+.optionsProClick ul li ul {
+	position:absolute;
+	left:0;
+	right:0;
+	border: 2px solid #f39d00;
+	margin-top: -2px;
+}
+
+.optionsProClick ul .clicker {
+	position:relative;
+	background:orange;
```

Comment:
Color `orange` should be written in hexadecimal form as `#ffa500`<br>Color literals like `orange` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Line should be indented with spaces, not tabs<br>Colon after property should be followed by one space